newIDE/app/src/ObjectsRendering/Renderers/CustomObjectLayoutingModel.js

```diff
@@ -85,43 +85,65 @@ const layoutFields = [
   'IsScaledProportionally',
 ];
 
+export const getProportionalPositionX = (positionName: string): ?number => {
+  const horizontalPositionName = (positionName.includes('-')
```

Comment:
can you add a small comment explaining why this check on "-" is needed? (this would help picturing what's going on) + same below

Reply:
+1